### ISSUE TITLE
New-SSHsession : sequence contains more than one element.

### ISSUE THREAD

**Issue by azam1980** [darkoperator/Posh-SSH#231]
Hello,
I have a server when trying to create a connection I am getting the error : New-SSHsession : sequence contains more than one element.  Till last week I was able to create the connection. 
Currently able to connect through putty. 
Verbose also not helping. Please advise, what else needs to be checked in order to debug this.
Able to login to the box using putty via ip and fqdn.

**Comment by darkoperator** [comment]
Was there any change on the server? What tupe of OS and version of SSH server is the server running ?

**Comment by azam1980** [comment]
No Change on the servers. OS is RHEL6 & 7 and on 7 - OpenSSH_7.4p1, OpenSSL 1.0.2k-fips
Posh SSH version I am using is 1.7.7

**Comment by darkoperator** [comment]
Will setup a lab machine this week to test, are you using username/pass, key, going through proxy?

**Comment by azam1980** [comment]
yes.  creating a creadential parameter using the username and password and using the parameter acceptkey.

new-sshsession -computername servername -credential $credentials -acceptkey

**Comment by darkoperator** [comment]
Can you move to the latest version?

```
PS C:\Users\carlos\Documents\WindowsPowerShell\Modules\Posh-SSH> $credobj = Get-Credential carlos
PS C:\Users\carlos\Documents\WindowsPowerShell\Modules\Posh-SSH> $credobj | Export-Clixml -Path ./carlos_test_cred.xml
PS C:\Users\carlos\Documents\WindowsPowerShell\Modules\Posh-SSH> New-SSHSession -ComputerName 192.168.1.169 -Credential (Import-Clixml .\carlos_test_cred.xml)

SessionId  Host
---------  ----
    1      192.168.1.169

```

**Comment by azam1980** [comment]
Thanks Carlos. Will do.
